.circleci/config.yml

```diff
@@ -43,6 +45,10 @@ jobs:
           command: |
             pre-commit run --all-files --show-diff-on-failure
 
+      - run:
+          name: Check python version is PEP 440 compliant
+          command: |
+            cd python && python -c "from packaging.version import Version;from tskit import _version; assert(str(Version(_version.tskit_version))==_version.tskit_version)"
```

Comment:
Looks good. I was imaging we just put it in as a standard unit test, though, and we can add ``packaging`` as a dev requirement. Would this not work?

Reply:
Yep that would work too, and I guess is more portable. I've updated the branch with the check done as a unit test.